screens/HealthProfessionalScreen.js

```diff
@@ -1,158 +1,131 @@
-import React, { useState } from 'react';
-import { View, Text, TextInput, TouchableOpacity, Dimensions, StyleSheet, Keyboard } from 'react-native';
-
+import React, { useState }from 'react';
+import { View, Text, TextInput, TouchableOpacity, Image, Button, Dimensions, StyleSheet, Keyboard } from 'react-native';
+import MenuIcon from '../assets/images/menu-icon';
 import * as Animatable from 'react-native-animatable';
-import { firebase } from '../components/Firebase/config';
-
-import Background from '../components/hpBackground';
-import NurseEllieLogo from '../assets/images/nurse-ellie-logo.svg';
```

Comment:
Use the NurseEllieLogo (SVG) for the logo for the screen, instead of the .png one.